workflows/phylorthology.nf

```diff
@@ -119,9 +119,11 @@ workflow PHYLORTHOLOGY {
     //
     // MODULE: Annotate UniProt Proteins
     //
-    ch_annotations = ANNOTATE_UNIPROT(ch_all_data.complete_prots)
+    //ch_all_data_ann = ch_download_annots.combine(ch_all_data.complete_prots)
+    ANNOTATE_UNIPROT(ch_all_data.complete_prots, params.download_annots)
         .cogeqc_annotations
         .collect()
+        .set { ch_annotations }
```

Comment:
....cause I felt like I should? I don't actually have strong feelings about this - seems like you tend to be inclined to just directly access the outputs from each module as needed rather than setting these as channels. Curious if there's an underlying reason for this? Could there be cases where outputting things like these into channels can introduce problems of their own?